Documentation~/menu.md

```diff
@@ -1,24 +1,172 @@
 # The ProBuilder menu
 
-Use the ProBuilder menu to access most of the ProBuilder editors, actions, tools, and a few special features that are only available through this menu. For example, you can repair and debug ProBuilder, or access the experimental [Boolean operations](boolean.md). 
-
-![The ProBuilder menu](images/pb_menu.png)
-
-The ProBuilder menu items include the following:
-
-* [ProBuilder Window](menu-open.md)
-* [Editors](menu-editors.md)
-* [Dimensions Overlay](menu-dimover.md)
-* [Selection](menu-selection.md)
-* [Interaction](menu-interaction.md)
-* [Object](menu-object.md)
-* [Geometry](menu-geometry.md)
-* [Materials](menu-materials.md)
-* [Vertex Colors](menu-vertexcolors.md)
-* [Experimental](menu-experimental.md) (only available when you enable the [experimental features preference](preferences.md#experimental))
-* [Repair](menu-repair.md)
-* [Export](menu-export.md)
-* [Debug](menu-debug.md)
-* [Actions](menu-actions.md)
+Use the ProBuilder menu to access most of the ProBuilder editors, actions, tools, and features that are only available through this menu. For example, you can repair and debug ProBuilder, or access the experimental [Boolean operations](boolean.md).
 
+## Editors
+
+Create new shapes and open editor windows.
+
+| **Menu item** | **Description** |
+| --- | --- |
+| **New Bezier Shape** | Use the **Bezier Shape** tool to define a Bezier curve around which ProBuilder extrudes a Mesh. **Warning:** This tool is experimental, meaning that it's still under development, and might reduce ProBuilder's stability. Use with caution. For more information, refer to [Bezier Shape tool](bezier.md). |
```

Comment:
```suggestion
| **New Bezier Shape** | Use the **Bezier Shape** tool to define a Bezier curve around which ProBuilder extrudes a Mesh. **Warning:** Use **New Bezier Shape** with caution. This tool is experimental and still under development. If you use this tool, it might reduce ProBuilder's stability. For more information, refer to [Bezier Shape tool](bezier.md). |
```